Cargo.toml

```diff
@@ -64,9 +64,9 @@ difference = "1.0"
 tempdir = "0.3"
 
 [features]
-default = []
-unstable = []
-dev = []
+default = ["syntax-highlight"]
```

Comment:
The CI should do a test build with default features disabled.

Also, the CI should no longer explicitly call out `syntax-highlight` if its enabled by default.

Reply:
Oh, I forgot to edit the CI scripts, going right now :)

Reply:
Oh, missed the deploy script.  Thanks for getting that.  So just .travis.yml and appveyor.yml left?

Reply:
appveyor has no mention of the feature, I've changed travis :)

Reply:
> appveyor has no mention of the feature, I've changed travis :)

My concern is with us accidentally breaking the no-default build.

> The CI should do a test build with default features disabled.

Reply:
hm, I don't know much about appveyor yet, what do you mean?

Reply:
This is a Cargo feature and not an appveyor vs travis thing.

`syntax-highlight` is a default feature, meaning cargo will auto-add it to the build when calling `cargo build`.  Users can disable this though, with `--no-default-features`.  We should make sure we don't break the build when no features are enabled.

So its just a matter of adding to both CIs
`cargo test --no-default-features`.

Reply:
I see! :D I will add this after fixing appveyor :)